medium-tests/src/test/java/mediumtest/hotspots/MatchWithServerHotspotsMediumTests.java

```diff
@@ -96,7 +96,7 @@ void it_should_track_local_only_hotspots() {
       .satisfies(result -> assertThat(result.getSecurityHotspotsByIdeRelativePath())
         .hasEntrySatisfying(Path.of("file/path"), hotspots -> {
           assertThat(hotspots).hasSize(1).allSatisfy(hotspot -> assertThat(hotspot.isRight()).isTrue());
-          assertThat(hotspots).usingRecursiveComparison().ignoringFields("right.id")
+          assertThat(hotspots).usingRecursiveComparison().ignoringFields("lsp4jEither.right.id")
```

Comment:
As I commented earlier, I don't like to expose this thing in tests. But I'm not sure there is a better way